kinetic/distribution.yaml

```diff
@@ -106,6 +106,22 @@ repositories:
       url: https://github.com/ros/actionlib.git
       version: indigo-devel
     status: maintained
+  adi_driver:
+    doc:
+      type: git
+      url: https://github.com/tork-a/adi_driver.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/tork-a/adi_driver-release.git
+      version: 1.0.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: Did you follow the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to grant access to ros-pull-request-builder to this repository?

Reply:
Sorry, accidentally the release repo was private when I did bloom release for kinetic firstly. Now the repo is public, how can I fix it?

Reply:
Excuse me again, I think I should not have set test_pull_requests for my package.
I hope now it works.